doc/write_your_own/intro.rst

```diff
@@ -1,7 +1,11 @@
 Writing Your Own Cube Browser
 =============================
 
-Description of how you can write notebooks using individual widgets (components) or a collection of them with connections.
+You can use the Cube Browser and IPython widgets packages to write your own Explorer notebooks which can offer as few or as many options and widgets as you wish.
+
```

Comment:
"which can offer as few or as many options and widgets as you wish".  I don't think you need the "as few" options as you are more likely to use the Browser.  It makes the sentence a little harder to read.